pkg/dao/repository_config_test.go

```diff
@@ -709,20 +738,28 @@ func (suite *RepositoryConfigSuite) TestListFilterMultipleVersions() {
 
 	quantity := 20
 
-	err := seeds.SeedRepositoryConfigurations(suite.tx, quantity,
-		seeds.SeedOptions{OrgID: orgID, Versions: &[]string{config.El7, config.El8, config.El9}})
-	assert.Nil(t, err)
+	assert.Nil(t, seeds.SeedRepositoryConfigurations(suite.tx, quantity/2,
+		seeds.SeedOptions{OrgID: orgID, Versions: &[]string{config.El7, config.El8, config.El9}}))
+
+	assert.Nil(t, seeds.SeedRepositoryConfigurations(suite.tx, quantity/2,
+		seeds.SeedOptions{OrgID: orgID, Versions: &[]string{config.El7}}))
 
 	//Seed data to a 2nd org to verify no crossover
-	err = seeds.SeedRepositoryConfigurations(suite.tx, quantity,
-		seeds.SeedOptions{OrgID: "kdksfkdf", Versions: &[]string{config.El7, config.El8, config.El9}})
-	assert.Nil(t, err)
+	assert.Nil(t, seeds.SeedRepositoryConfigurations(suite.tx, quantity,
+		seeds.SeedOptions{OrgID: "kdksfkdf", Versions: &[]string{config.El7, config.El8, config.El9}}))
 
 	response, count, err := GetRepositoryConfigDao(suite.tx).List(orgID, pageData, filterData)
 
 	assert.Nil(t, err)
 	assert.Equal(t, quantity, len(response.Data))
 	assert.Equal(t, int64(quantity), count)
+
+	//By setting SortBy to "version asc" we expect the first page versions lengths to be half and half or 1 and 3
```

Comment:
```suggestion
	// By setting the above seed values, we expect the first page to contain versions 10 versions of length 1 and 10 versions of length 3
```

Reply:
I added this to my current branch (Content-66), the commit suggestion option wasn't working for github for some reason @swadeley